pkg/features/pilot/pilot.go

```diff
@@ -118,12 +118,15 @@ var (
 		return time.Second * time.Duration(duration)
 	}
 
-	// EnableLocalityLoadBalancing provides an option to enable the LocalityLoadBalancerSetting feature
-	// as well as prioritizing the sending of traffic to a local locality. Set the environment variable to any value to enable.
-	// This is an experimental feature.
-	enableLocalityLoadBalancingVar = env.RegisterStringVar("PILOT_ENABLE_LOCALITY_LOAD_BALANCING", "", "")
-	EnableLocalityLoadBalancing    = func() bool {
-		return len(enableLocalityLoadBalancingVar.Get()) != 0
+	enableLocalityLoadBalancingVar = env.RegisterStringVar(
```

Comment:
should we just make this bool var?

Reply:
This is like an API change, so with others flags (bool and string abused ) we do it later?

Reply:
Good point. I am not against changing all of them to be bool in 1.3 (in another PR). What do you think

Reply:
See previous comment. Please move this to mesh config. This is a full feature, not an alpha thing. Asking people to kill and restart Pilot to turn this feature off is not good UX.. or asking them to hack the helm chart to add an env var to pilot is not great. Lets have this as a global setting in mesh config or under pilot - doesn't matter.